packages/javascript/src/__tests__/index.test.ts

```diff
@@ -23,4 +23,50 @@ describe("Appsignal", () => {
       expect(promise).resolves
     })
   })
+
+  describe("wrap", () => {
```

Comment:
Can you add a non-async function test? (just to cover our bases :))

Reply:
Is b894775 what you meant?